django-web-app/templates/register.html

```diff
@@ -88,10 +88,9 @@ <h1 class="head">Register</h1>
             </div>
         </div>
 
-        {% comment %} <div class="signup-img-box">
-            <img src="{% static 'img/signup-img.webp' %}" alt="Register CogniGuard">
-
-        </div> {% endcomment %}
+        <div class="signup-img-box">
+            <img src="{% static 'img/register.'jpg %}" alt="Register CogniGuard">
```

Comment:
There is a minor issue, see the static image link closing quotes '' 
Fix them. 
It is raising internal server error 500 ( see vercel preview)